js/main.js

```diff
@@ -25,4 +33,61 @@ Pineapples can be consumed fresh, cooked, juiced, or preserved. They are found i
 
 Strawberry
 The garden strawberry (or simply strawberry; Fragaria × ananassa) is a widely grown hybrid species of the genus Fragaria (collectively known as the strawberries). It is cultivated worldwide for its fruit. The fruit (which is not a botanical berry, but an aggregate accessory fruit) is widely appreciated for its characteristic aroma, bright red color, juicy texture, and sweetness. It is consumed in large quantities, either fresh or in such prepared foods as preserves, fruit juice, pies, ice creams, milkshakes, and chocolates. Artificial strawberry flavorings and aromas are also widely used in many products like lip gloss, candy, hand sanitizers, perfume, and many others.
-*/
\ No newline at end of file
+*/
+
+//this is my code//
+$(document).ready(function() {
+	
+
+ ///var fruit = $('fruit li').val();
+
+		//$('.fruit li').val();     
+       
+
+ 
+	 $('.fruit li').click(function() {
+	 		event.preventDefault();
+
+
+		$('.main li').removeClass('fruit active');
+
+		$(this).addClass('fruit active');	
+
+		$(this).addClass('results active');
```

Comment:
You got the right idea here but the only class you actually need to add/remove is `'active'`.